tests/e2e/pageobjects/ide/OpenWorkspaceWidget.ts

```diff
@@ -54,12 +54,16 @@ export class OpenWorkspaceWidget {
         await this.waitWidgetIsClosed();
     }
 
-    async expandTreeToPath(path: string) {
+    async expandTreeToPath(path: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT): Promise<any> {
```

Comment:
I might miss something... What's the point of returning a Promise here?
Shouldn't `pathNodes.map` be replaced by `pathNodes.forEach`?

Reply:
In the forEach loop as i remember `await` does not work properly. For avoiding problems with async. code we wrap it in the map and return promises. As second variant we can use other loop like `for`

Reply:
I confirm that I had problems with that.